src/schemas/types.js

```diff
@@ -1,20 +1,97 @@
+export const typeFormatChoices = [
+  {
+    'key': 'int32',
+    'label': '32-bit integer',
+    'conditions': {
+      '../type': 'integer'
+    }
+  },
+  {
+    'key': 'int64',
+    'label': '64-bit integer',
+    'conditions': {
+      '../type': 'integer'
+    }
+  },
+  {
+    'key': 'float',
+    'label': '32-bit floating point',
+    'conditions': {
+      '../type': 'number'
+    }
+  },
+  {
+    'key': 'double',
+    'label': '64-bit floating point',
+    'conditions': {
+      '../type': 'number'
+    }
+  },
+  {
+    'key': '',
```

Comment:
Should this key be empty? In general, *explicit is better than implicit*, so 'string' would be more explicit than ' '.